wordpress-mobile/WordPress-Android
# Stats revamp remove upgrade alert

This PR removes the upgrade alert popup (see image below) and related resources on Jetpack app

<img width=320 src=https://user-images.githubusercontent.com/990349/212582175-812fdd18-2c2a-4675-9319-e2ed8c810416.png />

Fixes #

To test:

- Launch Jetapack app
- Select a site that hasn't been already upgraded to new stats
- Navigate to Stats (Insights tab)
- Verify the upgrade alert doesn't popup (as shown above)
- Verify also that the new revamped stats are shown (Views & Visitors, Total Likes, Total Comments and Total Followers cards)

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Existing tests and manual tests

3. What automated tests I added (or what prevented me from doing so)
Existing unit tests

PR submission checklist:
